internal/server/models/encryption.go

```diff
@@ -17,18 +17,39 @@ var SymmetricKey *secrets.SymmetricKey
 // SkipSymmetricKey is used for tests that specifically want to avoid field encryption
 var SkipSymmetricKey bool
 
-func (s EncryptedAtRest) Value() (driver.Value, error) {
+func (s EncryptedAtRest) Encrypt() (string, error) {
```

Comment:
Are these encryption changes related to device flow or just done in passing?

Reply:
Was going to use this to encrypt the access key stored, but that causes problems when we switch to per-org encryption keys, so I ended up leaving it out. We could discard these changes or keep them, since it provides a manual way to encrypt and decrypt these fields, which we will likely need as we move away from gorm.